### PR TITLE
add es locale to build gha

### PR DESCRIPTION
**Pull request by gmanchon**
add support for `es` locale

the build will succeed once https://github.com/lewagon/data-setup/pull/191 and https://github.com/lewagon/setup/tree/setup-readme.es.md are merged

### CODE REVIEW

File: build.rb [hunk 1]
```diff
@@ -140,8 +140,7 @@
   python_checkup
 ]
 
-# LOCALES = ["", "es"]  # TODO: add support for the es locale when the content is merged
-LOCALES = [""]  # english locale only
+LOCALES = ["", "es"]  # english + spanish locales
```
Comment (gmanchon) [inline build.rb:143]:
```suggestion
LOCALES = ["", "es"]  # handles english + spanish locales
```

Comment (gmanchon) [reply build.rb]: the goal was to rerun the GHA

**Comment by gmanchon** [comment]
@krokrob following the merge of https://github.com/lewagon/data-setup/pull/191 we can tell the GHA to generate the es versions

=> then we will need to handle [desinstalar conda](https://github.com/lewagon/data-setup/pull/204) 😁

File: build.rb [hunk 2]
```diff
@@ -135,8 +135,7 @@
...
```
Comment (krokrob) [inline build.rb:138]: Why don't we specify `"en"`?

**Review by krokrob — COMMENTED** [review]
@gmanchon I have just 1 question

File: build.rb (hunk 2 above)
Comment (gmanchon) [reply build.rb:138]: `"en"` is specified by `""` since the `README.md` files have no extension for english content

**Pull request merged**